### PR TITLE
feat: borrow freeze warning

### PR DESCRIPTION
**Pull request by grothem**
Add a warning banner to alert users to recent proposal that will temporarily disable borrowing of ETH in the V2 Ethereum market.

Proposal: https://snapshot.org/#/aave.eth/proposal/0xa121311c67b7a5bbe5b8b5fe1911663a0ab94ed339a6a4b0e1b9443f670a0e97

### CODE REVIEW

File: src/layouts/AppHeader.tsx [hunk 1]
```diff
@@ -83,91 +83,120 @@ export function AppHeader() {
   );
```
Comment (grothem) [inline src/layouts/AppHeader.tsx:83]: The diff is ugly, but I wrapped the `HideOnScroll` component in a fragment, and then show the top banner if the mobile menu is not open

**Review by foodaka — DISMISSED** [review]

File: src/layouts/AppHeader.tsx [hunk 2]
```diff
@@ -83,91 +83,121 @@ export function AppHeader() {
...
+                <Button
+                  variant="surface"
+                  size="small"
+                  color="primary"
+                  sx={{
+                    backgroundColor: '#B6509E',
+                    '&:hover, &.Mui-focusVisible': { backgroundColor: 'rgba(182, 80, 158, 0.7)' },
+                  }}
+                >
+                  TESTNET
```
Comment (drewcook) [inline src/layouts/AppHeader.tsx:163]: Should this use `<Trans/>` tags?

File: src/layouts/AppHeader.tsx [hunk 3]
```diff
@@ -83,91 +83,121 @@ export function AppHeader() {
...
+            aria-label="Go to homepage"
+            sx={{
+              lineHeight: 0,
+              mr: 3,
+              transition: '0.3s ease all',
+              '&:hover': { opacity: 0.7 },
+            }}
+            onClick={() => setMobileMenuOpen(false)}
+          >
+            <img src={uiConfig.appLogo} alt="An SVG of an eye" height={20} />
```
Comment (drewcook) [inline src/layouts/AppHeader.tsx:149]: Could we use nextjs's `<Image />` component here?

Comment (grothem) [reply src/layouts/AppHeader.tsx:149]: This diff isn't very helpful, but this is existing. My changes are from lines 86 - 113

**Comment by grothem** [comment]
Closing this as we're getting close to when the proposal can be executed. We'll just skip the banner and add in the warning messages from #1089

**Pull request closed without merging**